static/src/stylesheets/module/content/_gallery.scss

```diff
@@ -245,3 +245,341 @@
         }
     }
 }
+
+// New gallery stuff
+
+// TODO: put in pasteup
+$multimedia-support-5: #161616;
```

Comment:
we're in the process of moving and documenting, but you can now just the value to `static/src/stylesheets/pasteup/palette/_src.scss`.

we need to define the relationship between `_vars` and `palette/_src.scss`, but we can take care of that